### ISSUE TITLE
ADuCM355 EIS example not running correctly

### ISSUE THREAD

**Issue by caiossoliveira** [analogdevicesinc/aducm355-examples#5]
Hello,
I am running the EIS Example from <https://www.analog.com/media/en/technical-documentation/user-guides/eval-aducm355qspz-ug-1308.pdf>, but instead of showing in the prompt to press the S2 switch to start the impedance test, it starts running without it. Besides that, instead of measuring in various frequencies, as shown in Figure 25, it only measures in 60000Hz. I solved the sweep issue by changing the _SweepEn_ parameter, but the signal visualized on the oscilloscope does not correspond to the frequency variation set in the code or from the prints in the prompt.

I do not know if the example code from here is different from the application note. 

Is there any advice about how to correct this issue to get the same code behavior as given in the application note example? And there is any code documentation available?

Thank you!

**Comment by r3410** [comment]
Hi caios,   I'm having the same problem, do you solve it?

**Comment by caiossoliveira** [comment]
> Hi caios, I'm having the same problem, did you solve it?
No, there is no routine to start the measurement after a button press event in the code. Also, I could never see a proper signal in the oscilloscope. I gave up using this board and finished the project with another one.

**Comment by alxtee** [comment]
Which other board did you end up using?

**Comment by caiossoliveira** [comment]
A STM32f4.

**Comment by Gopalkrishnbiradar** [comment]
I am currently working on this. Please let me know if any one has solution.
